### PR TITLE
Fix/code coverage

### PR DESCRIPTION
**Pull request by sverholen**

### CODE REVIEW

File: ldes-client/src/main/java/be/vlaanderen/informatievlaanderen/ldes/client/services/LdesFragmentFetcherImpl.java [hunk 1]
```diff
@@ -78,9 +74,7 @@ public LdesFragment fetchFragment(String fragmentUrl) {
                             }
                         });
             }
-        } catch (IOException e) {
-            LOGGER.error("An I/O exception occurred while fetching fragment {}", fragmentUrl, e);
```
Comment (Yalz) [inline ldes-client/src/main/java/be/vlaanderen/informatievlaanderen/ldes/client/services/LdesFragmentFetcherImpl.java:82]: so we no longer Log this problem?

File: ldes-workbench-nifi/ngsiv2-to-ngsi-ld-processor/src/main/java/be/vlaanderen/informatievlaanderen/ldes/processors/NgsiV2ToLdTranslatorDefaults.java [hunk 2]
```diff
@@ -1,14 +1,11 @@
...
 	private NgsiV2ToLdTranslatorDefaults() {}
 	
-	public static final String CONTENT_TYPE_JSON = "application/json";
-	
-	public static final String NIFI_TRUE = "true";
-	public static final String NIFI_FALSE = "false";
-
 	public static final String DEFAULT_CORE_CONTEXT = "https://uri.etsi.org/ngsi-ld/v1/ngsi-ld-core-context.jsonld";
 	public static final String LD_CONTEXT_WATER_QUALITY_OBSERVED = null;
 	public static final String DEFAULT_ADD_WKT_FOR_GEOJSON_PROPERTIES = NIFI_FALSE;
```
Comment (Yalz) [inline ldes-workbench-nifi/ngsiv2-to-ngsi-ld-processor/src/main/java/be/vlaanderen/informatievlaanderen/ldes/processors/NgsiV2ToLdTranslatorDefaults.java:11]: I see the value in unused.. we'll need to include the pmd plugin in this repo too :)

File: ldes-workbench-nifi/ngsiv2-to-ngsi-ld-processor/src/main/java/be/vlaanderen/informatievlaanderen/ldes/processors/NgsiV2ToLdTranslatorDefaults.java [hunk 3]
```diff
@@ -1,14 +1,11 @@
...
 	
 	private NgsiV2ToLdTranslatorDefaults() {}
 	
-	public static final String CONTENT_TYPE_JSON = "application/json";
-	
-	public static final String NIFI_TRUE = "true";
-	public static final String NIFI_FALSE = "false";
-
 	public static final String DEFAULT_CORE_CONTEXT = "https://uri.etsi.org/ngsi-ld/v1/ngsi-ld-core-context.jsonld";
 	public static final String LD_CONTEXT_WATER_QUALITY_OBSERVED = null;
```
Comment (Yalz) [inline ldes-workbench-nifi/ngsiv2-to-ngsi-ld-processor/src/main/java/be/vlaanderen/informatievlaanderen/ldes/processors/NgsiV2ToLdTranslatorDefaults.java:10]: Would generalise this. LD_CONTEXT_TARGET or something?

**Review by Yalz — APPROVED** [review]

File: ldes-workbench-nifi/ngsiv2-to-ngsi-ld-processor/src/main/java/be/vlaanderen/informatievlaanderen/ldes/processors/NgsiV2ToLdTranslatorDefaults.java (hunk 2 above)
Comment (sverholen) [reply ldes-workbench-nifi/ngsiv2-to-ngsi-ld-processor/src/main/java/be/vlaanderen/informatievlaanderen/ldes/processors/NgsiV2ToLdTranslatorDefaults.java:11]: These values were moved to the common module

File: ldes-client/src/main/java/be/vlaanderen/informatievlaanderen/ldes/client/services/LdesFragmentFetcherImpl.java (hunk 1 above)
Comment (sverholen) [reply ldes-client/src/main/java/be/vlaanderen/informatievlaanderen/ldes/client/services/LdesFragmentFetcherImpl.java:82]: The exception is thrown, it should be logged by nifi itself

**Pull request merged**